src/main/java/org/snomed/snowstorm/core/data/services/ServiceError.java

```diff
@@ -0,0 +1,11 @@
+package org.snomed.snowstorm.core.data.services;
```

Comment:
I created ServiceError so I could throw it In RelationshipService if there is a problem when reading MRCM data (line 95).

Reply:
I've deleted this and using RuntimeServiceException instead.

Reply:
It might be better to wrap the ServiceException.
throw new ServiceError("Trouble loading active MRCM data", e);

Reply:
Thanks for the suggestion - I have added that to the RuntimeServiceException.